### PR TITLE
Implements the Bookplate Model, ChangeSet, Decorator, and Controller

### LINKED ISSUE #1026: Create Bookplate model

### PR DESCRIPTION
**Pull request by jrgriffiniii**
In addition Modifies the routes and abilities; Restructures the "display" and "suppress" Decorator methods.  Resolves #1026

### CODE REVIEW

File: app/decorators/application_decorator.rb [hunk 1]
```diff
@@ -7,26 +7,27 @@ class ApplicationDecorator < Draper::Decorator
 
   # Add a set of attributes to be displayed
   # @param attribute_names [Symbol] the symbolized names of the attributes being displayed
-  def self.display(attribute_names)
-    self.displayed_attributes += Array.wrap(attribute_names)
+  def self.display(*attribute_names)
+    self.displayed_attributes += Array.wrap(attribute_names.flatten)
```
Comment (hackmastera) [inline app/decorators/application_decorator.rb]: Can you say why you did this splat / flatten business?

Comment (hackmastera) [reply app/decorators/application_decorator.rb]: oh i see it just makes the method invocation look pretty

File: app/decorators/bookplate_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          term.definition.html_safe
+        end +
+        h.content_tag("p") do
+          I18n.t("valhalla.works.show.attributes.rights_statement.boilerplate").html_safe
+        end
+    end
+  end
+
+  def manageable_structure?
+    true
```
Comment (hackmastera) [inline app/decorators/bookplate_decorator.rb]: i think this should be false? we left out a bunch of structure stuff

**Comment by hackmastera** [comment]
I'd like to try reorganizing the commits so that all the refactors and extractions come before the addition of the new model, so that commit can serve as a clean source of documentation on adding new models.

**Comment by escowles** [comment]
@hackmastera: 👍 that seems like a good way to make documenting this process easier

**Comment by hackmastera** [comment]
please close in favor of #1043 if it looks good to you

**Comment by escowles** [comment]
Closed in favor of https://github.com/pulibrary/figgy/pull/1045

**Pull request closed without merging**